kiosk/src/Components/AddingGame.tsx

```diff
@@ -47,6 +51,22 @@ const AddingGame: React.FC<IProps> = ({ kiosk }) => {
         return true;
     }
 
+    const getGameNames = (gameList: GameData[]): string[] => {
+        const gameNames = [];
+        for (const game of gameList) {
+            gameNames.push(game.name);
+        }
+        return gameNames;
+    }
+
+    const displayGamesAdded = (addedGames: GameData[]): void => {
+        const gameNames = getGameNames(addedGames);
+        const games = gameNames.toString();
```

Comment:
```suggestion
        const games = gameNames.join(", ");
```
Since this will be displayed to the user, a comma-delimited list might be more human-readable.